Mathlib/Algebra/Algebra/Basic.lean

```diff
@@ -132,18 +132,18 @@ end CommSemiring
 
 section Ring
 
-variable [CommRing R]
-variable (R)
```

Comment:
Can you explain this change?

Reply:
I think this is not needed, since the newly added instance has `{R : Type*}` which will override the existing `R`.

Reply:
I thought `[CommRing R]` might interfere with the new instance so I moved it down. It is unnecessary if no confusion will be caused.